nav2_smac_planner/src/a_star.cpp

```diff
@@ -273,6 +335,9 @@ bool AStarAlgorithm<NodeT>::createPath(
     return false;
   }
 
+  NodeVector coarse_list, fine_list;
+  _goal_manager.prepareGoalsForExpansion(coarse_list, fine_list, _coarse_search_resolution);
```

Comment:
```suggestion
  _goal_manager.prepareGoalsForAnalyticExpansion(coarse_list, fine_list, _coarse_search_resolution);
```

Also rename this to be clear about what expansion

Reply:
done